### DISCUSSION TITLE
Unset DB_PASSWORD

### DISCUSSION THREAD

**Discussion by soudaburger** [firefly-iii/firefly-iii#8940] [General]
I am trying to leverage deploying FF into Google Cloud GKE (Kubernetes) using workload identity. As such, I don't actually need to set a password. It's not that you can just your password to "", you're actually not supposed to send a password to the DB at all. Is there a way to do this with FF? Any help here would be great. Thanks!

**Comment by JC5** [comment]
As far as I know, omitting the DB_PASSWORD will mean it doesn't get sent to the database. Are you seeing something else?
